src/rezplugins/build_system/cmake.py

```diff
@@ -75,8 +75,8 @@ def bind_cli(cls, parser):
                             type=str, choices=cls.build_targets,
                             default=settings.build_target,
                             help="set the build target (default: %(default)s).")
-        parser.add_argument("--bs", "--build-system", dest="build_system",
-                            type=str, choices=cls.build_systems.keys(),
+        parser.add_argument("--bs", "--cmake-build-system",
```

Comment:
this needs 
`dest="cmake_build_system"`,

If not when running with --bs it won't be picked in line 96